.github/workflows/pages.yml

```diff
@@ -49,13 +49,15 @@ jobs:
         env:
           JEKYLL_ENV: production
       - name: Upload artifact
+        if: ${{ github.event_name == 'push' && github.repository == 'GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration' && github.ref_name == 'default'}}
```

Comment:
this will match branches named `default`. We should probably use: `github.event.repository.default_branch` (without `'`)

Reply:
Done